### DISCUSSION TITLE
‌ে-কার যুক্ত হরফের পর  ্য (য-ফলা) লেখা যায়না #fixed

### DISCUSSION THREAD

**Discussion by rashidi77** [OpenBangla/OpenBangla-Keyboard#261] [General]
ফিক্সড লেআউট দিয়ে ওল্ড স্টাইলে টাইপিং এর ক্ষেত্রে এ-কার যুক্ত হরফের পর য-ফলা যুক্ত করা যায় না। বরং  য-ফলাটি ্+য তে পরিবর্তিত হয়ে যায়।

উদাহরণস্বরূপ : ‘সাহায্য‌ের’ লিখতে চাইলে ‘সাহাযে্যর’ হয়ে যায়।

সাময়িক সমাধান :  হরফ+্য+‌ে+Whitespace character এভাবে লেখা। 

Tested on : OpenBangla Keyboard 3.00

**Comment by mominul** [comment]
বাগটি ফিক্স করা হয়েছে, আপনাকে নতুন করে ক্লোন আপডেট ও কম্পাইল করে ইন্সটল করতে হবে।

**Reply by rashidi77** [reply]
> বাগটি ফিক্স করা হয়েছে, আপনাকে নতুন করে ক্লোন আপডেট ও কম্পাইল করে ইন্সটল করতে হবে।

আপনাদের অসংখ্য ধন্যবাদ এতো দ্রুত সমস্যাটির সমাধানের জন্য।

**Reply by mominul** [reply]
ধন্যবাদ আপনাকেও! তবে বিশেষ ধন্যবাদ @NabilSnigdho  কে, তাড়াতাড়ি ফিক্স পাঠানোর জন্য! :blush:
